### PR TITLE
fix: Fixed Shimmer Layout Issue

### LINKED ISSUE #1419: Shimmer Not Properly Visible in Landscape Mode
**Describe the chore**
The shimmer to display tickets in Landscape Mode is not proper.
Adjust It's width.

Screenshot:
Screenshot_20190325-201417

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1419 

Changes: Now shimmer is visible perfectly

Screenshots for the change:
Screenshot_20190326-084538

### CODE REVIEW

**Comment by somenath1435** [comment]
@liveHarshit please review

**Comment by somenath1435** [comment]
@nikit19 @somenath1435 PR is updated

**Comment by somenath1435** [comment]
@iamareebjamal @liveHarshit please review

File: app/src/main/res/layout/placeholder_item_card_tickets.xml [hunk 1]
```diff
@@ -10,57 +10,74 @@
...
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/layout_margin_none"
-                android:layout_weight="1"
-                android:background="@color/shimmer_background"
-                android:ellipsize="end" />
+        <View
+            android:id="@+id/view2"
+            android:layout_width="0dp"
+            android:layout_height="30dp"
```
Comment (liveHarshit) [inline app/src/main/res/layout/placeholder_item_card_tickets.xml]: Define all dimensions in dimenstion resources file.

Comment (somenath1435) [reply app/src/main/res/layout/placeholder_item_card_tickets.xml]: @liveHarshit Done

File: app/src/main/res/layout/placeholder_item_card_tickets.xml [hunk 2]
```diff
@@ -10,57 +10,74 @@
...
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/layout_margin_none"
-                android:layout_marginBottom="@dimen/layout_margin_small"
-                android:layout_weight="1"
-                android:background="@color/shimmer_background"
-                android:ellipsize="end" />
+        <View
+            android:id="@+id/view1"
+            android:layout_width="@dimen/layout_margin_none"
+            android:layout_height="@dimen/view_height_30dp"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/placeholder_item_card_tickets.xml]: Poor dimen name

File: app/src/main/res/layout/placeholder_item_card_tickets.xml [hunk 3]
```diff
@@ -10,57 +10,74 @@
...
+            android:background="@color/shimmer_background"
+            android:ellipsize="end"
+            app:layout_constraintBottom_toTopOf="@+id/view3"
+            app:layout_constraintLeft_toRightOf="@+id/viewLeft"
+            app:layout_constraintRight_toLeftOf="@+id/viewRight"
+            app:layout_constraintTop_toBottomOf="@+id/view1" />
 
-        </LinearLayout>
+        <View
+            android:id="@+id/view3"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/placeholder_item_card_tickets.xml]: Poor ids

**Comment by somenath1435** [comment]
@iamareebjamal PR is updated.

**Comment by somenath1435** [comment]
@iamareebjamal Any other changes required?

**Pull request merged**